.travis.yml

```diff
@@ -1,6 +1,18 @@
 language: rust
 rust:
     - nightly
+
+before_script:
+    - "sh -e /etc/init.d/xvfb start"
+    - "export DISPLAY=:99.0"
+    - "wget http://selenium-release.storage.googleapis.com/2.50/selenium-server-standalone-2.50.1.jar"
+    - "java -jar selenium-server-standalone-2.50.1.jar > /dev/null &"
+    - sleep 5
+    - nvm install 4.2
+    - nvm use 4.2
+    - npm install selenium-webdriver
+
 script:
     - cargo build
     - RUST_BACKTRACE=1 cargo test
+    - cargo run > /dev/null & node test/selenium/ftu_test.js
```

Comment:
That syntax errors with `unexpected token ;`, but changed it to:
`(cargo run > /dev/null &) ; node test/selenium/ftu_test.js`